sample.py

```diff
@@ -0,0 +1,107 @@
+"""sample.py uses models generated by training.py and
+generates a sample output based on it. The sample is saved
+in the /sample directory.
+"""
+
+import os
+import sys
+import glob
+from pathlib import Path
+from subprocess import Popen
+
+print('sample.py takes a model and generates sample output.')
+print('If you do not have a model file, close and run training.py')
+print('Click enter to continue or "q" to exit.')
+option = input(': ')
+print()
+
+# pull model name and numer of epochs from jsoon/csv/yaml file
+# have it list and allow the user to choose which one they would
+# like to sample
+# someone determine the length of the sample file
+# save to sample in sample/ or something
+
+if not option == 'q':
+    print('Tpye in the name of the model.')
```

Comment:
HA HA HA HA YOU MADE A TYPO YOU MADE A TYPO

Reply:
Fixed 8cc6db882ff4fbaa01f99a576265f8c130545181